remote.go

```diff
@@ -1124,6 +1134,35 @@ func (wd *remoteWD) Screenshot() ([]byte, error) {
 	return ioutil.ReadAll(decoder)
 }
 
+func (wd *remoteWD) WaitWithTimeoutAndInterval(condition Condition, timeout time.Duration, interval time.Duration) (error) {
```

Comment:
Collapse the common types of the two arguments together:

```go
(condition Condition, timeout, interval time.Duration)
```

Fix in the interface definition as well.